server/testutil/testauth/testauth.go

```diff
@@ -101,11 +102,13 @@ func (a *TestAuthenticator) AuthenticateHTTPRequest(w http.ResponseWriter, r *ht
 
 func (a *TestAuthenticator) AuthenticateGRPCRequest(ctx context.Context) context.Context {
 	if grpcMD, ok := metadata.FromIncomingContext(ctx); ok {
-		headerVals := grpcMD[TestApiKeyHeader]
-		for _, headerVal := range headerVals {
-			user, ok := a.testUsers[headerVal]
-			if ok {
-				return context.WithValue(ctx, testAuthenticationHeader, user)
+		for _, h := range []string{TestApiKeyHeader, jwtHeader} {
+			headerVals := grpcMD[h]
+			for _, headerVal := range headerVals {
+				user, ok := a.testUsers[headerVal]
+				if ok {
```

Comment:
you could inline this:

if user, ok := a.testUsers[headerVal]; ok {
  return context.WithValue(ctx, testAuthenticationHeader, user)
}

Reply:
Done